examples/water_boiler/water_boiler.py

```diff
@@ -58,4 +60,7 @@ def update(self, boiler_power, dt):
     plt.xlabel('time')
     plt.ylabel('temperature')
     plt.legend()
-    plt.show()
+    if os.getenv("NO_DISPLAY"):
```

Comment:
Please use `'` instead of `"` to follow the style of the rest of this file and project.